ReactNativeClient/lib/components/global-style.js

```diff
@@ -19,6 +19,7 @@ const globalStyle = {
 	disabledOpacity: 0.2,
 	colorUrl: '#7B81FF',
 	textSelectionColor: '#0096FF',
+	keyboardAppearance: 'light',
```

Comment:
I've been thinking it would be nice if a theme could specify whether it is "dark" or "light", as that could be useful in other places of the UI. So please could you do the following?

- In `global-style`, rename your variable "keyboardAppearance" to "appearance", which can be either "light" or "dark" as you have now
- In "addExtraStyles()", create the "keyboardAppearance" variable that's derive from the "appearance" one. So for now it will simply be `style.keyboardAppearance = style.appearence`

And the rest of the code would be unchanged.